test/bench_internal/parser_benchmarks.cpp

```diff
@@ -100,4 +100,4 @@ static void parse_string(benchmark::State& state)
 
     state.SetItemsProcessed(static_cast<int64_t>(size));
 }
-BENCHMARK(parse_string)->RangeMultiplier(2)->Range(16, 4 * 1024);
+BENCHMARK(parse_string)->RangeMultiplier(2)->Range(16, 4096);
```

Comment:
Is this change needed after `-bugprone-implicit-widening-of-multiplication-result` ?

Reply:
No.

Reply:
why change then?